rules/set-hostprocess-true/raw.rego

```diff
@@ -0,0 +1,41 @@
+package armo_builtins
+
+import future.keywords.if
+
+# Rule to check if container is set as a 'Host Process' container
+deny[msg] {
+    # Verify the object kind
+    obj := input[_]
+    allowed_kinds := {"Pod", "Deployment", "ReplicaSet", "DaemonSet", "StatefulSet", "Job", "CronJob"}
+    allowed_kinds[obj.kind]
+
```

Comment:
Please add OS check using the label `kubernetes.io/os: "windows"` located in `Node` object.